tests/test_app.py

```diff
@@ -37,13 +39,37 @@ def test_register_proposal_successful(self):
         node = Node.query.get(public_address)
         self.assertEqual(self.REMOTE_ADDR, node.ip)
 
+    def test_register_proposal_successful_with_dummy_type(self):
```

Comment:
can test_register_proposal_successful() test dummy service_type? In that case another one test test_register_proposal_successful_with_dummy_type will not be required

Reply:
I want to test both types, openvpn and dummy - i think it's best to leave them as separate tests, since mushing them together would end up in a single, long test that would test 2 different cases